### PR TITLE
Changed ToggleTerm keymaps

### PR DESCRIPTION
**Pull request by neutonfoo**
The keymap as noted on the docs on https://astronvim.github.io/Basic%20Usage/walkthrough and https://astronvim.github.io/Basic%20Usage/mappings for toggling the terminal is listed as `Ctrl + \`. However, in [mappings.lua](https://github.com/AstroNvim/AstroNvim/blob/main/lua/core/mappings.lua#L188-L191), they are instead `Ctrl + '`.

This PR changes the keymaps listed on the website to state `Ctrl + '`.

### CODE REVIEW

File: versioned_docs/version-2.4.0/Basic Usage/walkthough.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ To toggle file explorer you need to press `Space` + `e`
 
 ### Opening terminal
 
-To toggle terminal you need to press `Ctrl` + `\`
+To toggle terminal you need to press `Ctrl` + `'`
```
Comment (mehalter) [inline versioned_docs/version-2.4.0/Basic Usage/walkthough.md:13]: Could you please change this to `F7` for maximum compatibility in the walkthrough?

**Review by mehalter — CHANGES_REQUESTED** [review]
I made one comment and also could you please delete the `yarn.lock` file and revise the commit. Thanks for this contribution!

File: versioned_docs/version-2.4.0/Basic Usage/walkthough.md (hunk 1 above)
Comment (neutonfoo) [reply versioned_docs/version-2.4.0/Basic Usage/walkthough.md]: Done!

**Review by mehalter — APPROVED** [review]
LGTM

**Pull request merged**